lib/modules/dosomething/dosomething_signup/dosomething_signup.forms.inc

```diff
@@ -98,11 +98,14 @@ function dosomething_signup_form_submit($form, &$form_state) {
   }
 
   if (variable_get('rogue_collection', FALSE)) {
+    $form_state['values']['source'] = 'phoenix-web';
```

Comment:
oh that is a good question! I just noticed that it was saving in the database here at `null` when being forwarded to Rogue so wanted to make that more specific. I can delete this line though @DFurnes